modules/oadp-velero-cpu-memory-requirements.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-velero-cpu-memory-requirements_{context}"]
+= Velero CPU and memory requirements based on collected data
+
+The following recommendations are based on observations of performance made in the scale and performance lab. The backup and restore resources can be impacted by the type of plugin, the amount of resources required by that backup or restore, and the respective data contained in the persistent volumes (PVs) related to those resources.
+
+
+== CPU and memory requirement for configurations
+|===
+|Configuration types | ^[1]^ Average usage |^[2]^ Large usage |resourceTimeouts
+
+|CSI
+|Velero:
+
+CPU- Request 200m, Limits 1000m
+
+Memory - Request 256Mi, Limits 1024Mi
+
+|Velero:
+
+CPU- Request 200m, Limits 2000m
+
+
+Memory- Request  256Mi, Limits 2048Mi
+
+|
+
+|Restic
+|^[3]^ Restic:
+
+CPU- Request 1000m, Limits 2000m
+
+Memory - Request 16Gi, Limits 32Gi
+
+
+|^[4]^ Restic:
+
+CPU - Request 2000m, Limits 8000m
+
+
+Memory - Request 16Gi, Limits 40Gi
+
+
+|900m
+
+
+
+|^[5]^ DataMover
+|
+|
+|10m - average usage
+
+60m - large usage
+|===
+
+[.small]
+--
+1. Average usage - use these settings for most usage situations.
```

Comment:
I think you may be trying to use Footnotes here, which our [contributing guidance](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#footnotes) discourages. Instead, would it be possilbe to use callouts? I've never tried to use them in a table, only in code samples. Maybe try replacing the [1]-[3] with <1>-<3> to form callouts?

Reply:
I tried the callouts in this table and it didn't work. I used the footnotes, because the other table that I found in this guide used these types of footnotes, so it will be consistent.

Reply:
@CarmiWisemon tbh, footnotes are OK. I'm not even sure why we are encouraged to avoid, but your use of them is perfectly fine here.